### PR TITLE
Update DESIGN_GUIDELINES.md

### PR DESCRIPTION
**Pull request by biffgaut**
*Issue #, if available:*
Update Design Guidelines to reflect changes in Events (non default Event Buses) and addition of WAF WebACLs

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 1c28e85720aa808be76e2432ba0cc04d21965c72
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=87b3380b-2ac1-4b19-91c5-2a98baa1b5d9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: DESIGN_GUIDELINES.md [hunk 1]
```diff
@@ -119,6 +119,21 @@ Existing Inconsistencies would not be published, that’s for our internal use 
 | apiGatewayRole	| iam.Role	||
 
 
+## WAF WebACL
+**Required Attributes on Props** 
+
+| Name    | Type     | Notes    |
+| --- | --- | --- |
+| existingWebaclObj? | waf.CfnWebACL	||
+| webaclProps?	| waf.CfnWebACLProps	||
```
Comment (hnishar) [inline DESIGN_GUIDELINES.md:128]: Are these supposed to be `aws_wafv2.CfnWebACLProps` & `aws_wafv2.CfnWebACL` ?

Comment (mickychetta) [reply DESIGN_GUIDELINES.md:128]: Yes they are from wafv2 but were imported as waf, will update for clarity

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 2ffd72128400786ee7d2859e58c0735ac0350c86
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=ee2351b7-8bc2-4f6b-b175-1a69add66b0a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by mickychetta — APPROVED** [review]

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 5c02db163e2a79271eb21c7eaf254f695f6212e9
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6fa423a9-d845-4f39-96ac-ece882294c43%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 057d5880e97e2be037e5be83f2b6ac8efb91383e
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=880ebedd-b902-4c59-af51-678ecf716ea4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 057d5880e97e2be037e5be83f2b6ac8efb91383e
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=b1ce7b66-880f-4491-8217-7dc2e6d15c26%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by biffgaut** [comment]
Doc only change - forcing merge.

**Pull request merged**